### ISSUE TITLE
Service log does not allow use of templates with blank service type

### ISSUE THREAD

**Issue by leducvin** [qatrackplus/qatrackplus#614]
Hi Randle, I found I can create service templates with blank service type (this seems to be allowed), but actually cannot use them.

Here is service template creation:

![image](https://user-images.githubusercontent.com/22982038/165083783-912323b5-25de-4743-8929-46495ee9dd1b.png)

Service event form (need to keep service type drop-down selection to blank for my template to be selectable):

![image](https://user-images.githubusercontent.com/22982038/165084180-54cfa442-ee80-4d6e-9d64-87234b42c200.png)

Upon clicking Save button, says Service Type is required, yet I need _not_ to select one for my template to be selectable:

![image](https://user-images.githubusercontent.com/22982038/165084390-d5796a97-aae5-47c0-966a-3361c4c27680.png)

So bottom line is I cannot save my service event. I'm on v3.1.0.1.
